Makefile

```diff
@@ -992,6 +1114,10 @@ open_write.o: \
 compile open_write.c open.h
 	./compile open_write.c
 
+package: \
+it man
+	./instpackage
+
```

Comment:
This is great, and gets me thinking about install documentation. The netqmail folks did us the favor of deleting everything in `INSTALL`, leaving nothing but a link to LWQ. I'd say the filename is ours to use.